pisa/analysis/analysis.py

```diff
@@ -762,6 +798,8 @@ def fit_hypo_inner(self, data_dist, hypo_maker, metric, minimizer_settings,
 
         # Record the Asimov distribution with the optimal param values
         hypo_asimov_dist = hypo_maker.get_outputs(return_sum=True)
+        gen_poisson_dist = hypo_maker.get_outputs(return_sum=False, force_standard_output=False)
```

Comment:
what does "gen" stand for? My immediate guess would be "generated".....but knowing your PR it's probably "generalized"? no need to abbreviate.....I'd rather have longer names that are clear

Reply:
Also, are these outputs now always retrieved, even if I fit a simple TS? (I'm a bit concerned about speed here)